internal/cluster/cluster.go

```diff
@@ -365,11 +364,6 @@ func (m *Manager) VerifyClusterUpdatability(c *common.Cluster) (err error) {
 	return err
 }
 
-func (m *Manager) SetGeneratorVersion(c *common.Cluster, version string, db *gorm.DB) error {
-	return db.Model(&common.Cluster{}).Where("id = ?", c.ID.String()).
-		Update("ignition_generator_version", version).Error
```

Comment:
don't you want to remove ignition_generator_version from cluster definition in cluster also?

Reply:
if you mean from swagger, see my answer to @filanov above